etc/crontab.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
+    <group id="default">
+        <job name="deactivate_quote" instance="Bolt\Boltpay\Cron\DeactivateQuote" method="execute">
+            <schedule>0 2 * * *</schedule>
```

Comment:
Once a day at 2 am. Not sure this is frequent enough. I'd set it once every hour.